### PR TITLE
:sparkles: added customs to shipment

### PR DESCRIPTION
**Pull request by M4tini**

### CODE REVIEW

File: specification/definitions/Shipment.json [hunk 1]
```diff
@@ -152,6 +152,82 @@
             }
           }
         },
+        "customs": {
+          "type": "object",
+          "properties": {
+            "package_type": {
```
Comment (SAMDevelopment) [inline specification/definitions/Shipment.json]: Services hebben bij ons ook al een property `package_type`. Misschien verwarrend?

`content_type`?

File: specification/definitions/Shipment.json [hunk 2]
```diff
@@ -152,6 +152,82 @@
...
+              "enum": [
+                "merchandise",
+                "sample_merchandise",
+                "returned_merchandise",
+                "documents",
+                "gifts"
+              ],
+              "example": "merchandise"
+            },
+            "invoice": {
```
Comment (SAMDevelopment) [inline specification/definitions/Shipment.json]: Misschien is `invoice` niet beschrijvend genoeg.

`invoice_number` of `invoice_id`?

File: specification/definitions/Shipment.json [hunk 3]
```diff
@@ -152,6 +152,82 @@
...
+                ],
+                "additionalProperties": false,
+                "properties": {
+                  "sku": {
+                    "type": "string",
+                    "example": "123456789"
+                  },
+                  "description": {
+                    "type": "string",
+                    "example": "Kittens"
```
Comment (SAMDevelopment) [inline specification/definitions/Shipment.json]: :open_mouth: 

Misschien hier iets realistischers in zetten dat je op de post doet. :stuck_out_tongue_closed_eyes: 

`xbox one x`?

File: specification/definitions/Shipment.json [hunk 4]
```diff
@@ -152,6 +152,82 @@
...
+                  },
+                  "item_value": {
+                    "$ref": "../includes/price.json"
+                  },
+                  "quantity": {
+                    "type": "string",
```
Comment (SAMDevelopment) [inline specification/definitions/Shipment.json]: `integer`?

Zodat we zeker weten dat we er mee kunnen rekenen?

File: specification/definitions/Shipment.json [hunk 5]
```diff
@@ -152,6 +152,82 @@
...
+            },
+            "non_delivery": {
+              "type": "string",
+              "enum": [
+                "return",
+                "abandon"
+              ],
+              "example": "return"
+            },
+            "incoterm": {
```
[the remark this answers is not in the document]
Comment (M4tini) [reply specification/definitions/Shipment.json]: I wanted to name it `duty` with enum `paid` `unpaid` but Lars said the current implementation is an industry standard.

[the remark this answers is not in the document]
Comment (SAMDevelopment) [reply specification/definitions/Shipment.json]: :nauseated_face:

File: specification/definitions/Shipment.json [hunk 6]
```diff
@@ -152,6 +152,86 @@
...
+                    "example": 2
+                  },
+                  "hs_code": {
+                    "type": "string",
+                    "minLength": 1,
+                    "example": "85171200"
+                  },
+                  "origin_country_code": {
+                    "type": "string",
+                    "minLength": 1,
```
Comment (Shodske) [inline specification/definitions/Shipment.json]: We could do our `country_code` validation here by using a pattern (or enum):
`^AD|AE|AF|AG|AI|AL|AM|AO|AQ|AR|AS|AT|AU|AW|AX|AZ|BA|BB|BD|BE|BF|BG|BH|BI|BJ|BL|BM|BN|BO|BQ|BR|BS|BT|BV|BW|BY|BZ|CA|CC|CD|CF|CG|CH|CI|CK|CL|CM|CN|CO|CR|CU|CV|CW|CX|CY|CZ|DE|DJ|DK|DM|DO|DZ|EC|EE|EG|EH|ER|ES|ET|FI|FJ|FK|FM|FO|FR|GA|GB|GD|GE|GF|GG|GH|GI|GL|GM|GN|GP|GQ|GR|GS|GT|GU|GW|GY|HK|HM|HN|HR|HT|HU|ID|IE|IL|IM|IN|IO|IQ|IR|IS|IT|JE|JM|JO|JP|KE|KG|KH|KI|KM|KN|KP|KR|KW|KY|KZ|LA|LB|LC|LI|LK|LR|LS|LT|LU|LV|LY|MA|MC|MD|ME|MF|MG|MH|MK|ML|MM|MN|MO|MP|MQ|MR|MS|MT|MU|MV|MW|MX|MY|MZ|NA|NC|NE|NF|NG|NI|NL|NO|NP|NR|NU|NZ|OM|PA|PE|PF|PG|PH|PK|PL|PM|PN|PR|PS|PT|PW|PY|QA|RE|RO|RS|RU|RW|SA|SB|SC|SD|SE|SG|SH|SI|SJ|SK|SL|SM|SN|SO|SR|SS|ST|SV|SX|SY|SZ|TC|TD|TF|TG|TH|TJ|TK|TL|TM|TN|TO|TR|TT|TV|TW|TZ|UA|UG|UM|US|UY|UZ|VA|VC|VE|VG|VI|VN|VU|WF|WS|YE|YT|ZA|ZM|ZW$`

File: specification/definitions/Shipment.json [hunk 7]
```diff
@@ -152,6 +152,86 @@
...
+                  "item_value": {
+                    "$ref": "../includes/price.json"
+                  },
+                  "quantity": {
+                    "type": "number",
+                    "example": 2
+                  },
+                  "hs_code": {
+                    "type": "string",
+                    "minLength": 1,
```
Comment (Shodske) [inline specification/definitions/Shipment.json]: HS codes are composed of numbers and periods (for as far as I can see), so we could add a pattern here:
`^[0-9\.]+$`

**Pull request merged**